internal/io/writer.go

```diff
@@ -82,7 +83,7 @@ func (w *PanicWriter) WriteNewLine() int {
 // character after the given string, and returns the number of bytes that were
 // written. It'll panic if any error occurs during writing.
 func (w *PanicWriter) WriteStringLine(p string) int {
-	return w.WriteString(p) + w.WriteNewLine()
+	return w.WriteString(html.UnescapeString(p)) + w.WriteNewLine()
```

Comment:
Hmm. I see what this is intending, but HTML, especially format-specific structures, should _never_ be in the fetched data from the source, regardless of the output style. Even if we were to have a non-console writer (like something that maybe exported the data into an HTML page or something), the structuring would be done in a separate place.

I guess what I'm trying to say is that HTML escaping shouldn't be done at the writer level, here, but instead should be fixed at the specific source that's failing to clean the fetched data. Does that make sense?

For example, the `webster` source currently attempts to clean certain strings:
https://github.com/Rican7/define/blob/00c33da75ed272debbe7f2ed3b6bdd10043ff9c0/source/webster/webster.go#L451-L454

We should do cleaning/sanitizing of sorts in the source, not the writer.

Reply:
@Rican7 what's the point of sanitizing and then unescaping? Don't those cancel each other out?